README.md

```diff
@@ -33,12 +33,12 @@ If no puzzle file is specified, `puzzle_files/example1.txt` is used by default.
 There are three constraints used in formulating this problem:
 1. Minimize the difference between squares of left-hand side and right-hand side of equation (minimum of zero).
```

Comment:
How is bullet 1 a constraint? As described here, it looks like an objective.

Reply:
You are right, it is an objective

Reply:
Updated the wording and distinguished objective/constraints.